index.html

```diff
@@ -3325,6 +3323,48 @@ <h2>
       <h2>
         Security Considerations
       </h2>
+      <section class="informative">
+        <h2 id="sec-enhancements">
+          Mechanisms to enhance security
+        </h2>
+        <p>
+          This specification includes a number of mechanisms to enhance the
+          security of the API:
+        </p>
+        <ul data-link-for="PaymentRequest">
+          <li>Explicit user action is required to call <a>PaymentRquest</a>'s
+          <a>show()</a> method. This reduces content's ability to unexpectedly
+          cause a request for payment to be displayed to the end-user.
+          </li>
+          <li>Interfaces are <a data-cite=
+          "WEBIDL#dfn-available-only-in-secure-contexts">available only in a
+          secure context</a>, to reduce the possibly that credentials will be
+          acquired and transferrer through insecure communication protocols.
+          </li>
+          <li>A <a>top-level browsing context</a> need to explicitly grant an
+          <a>iframe</a> the ability to access the <a>PaymentRequest</a>
+          interface via the <a>allowpaymentrequest</a> attribute. This prevents
+          embedded third-party content from accessing the interfaces of the
+          <cite>Payment Request API</cite> without the <a>top-level browsing
+          context</a>'s permission.
+          </li>
+          <li>In the definition of <a>canMakePayment()</a> the Working Group
```

Comment:
I still think this remains an open issue, wrt @lknik's findings in https://github.com/w3c/payment-request/issues/641.

Reply:
It does seem to me that we still have some privacy holes here. I will commit to submitting a separate issue about it.